test_programs/execution_success/signed_div/src/main.nr

```diff
@@ -4,7 +4,16 @@ struct SignedDivOp {
     result: i8,
 }
 
-unconstrained fn main(ops: [SignedDivOp; 15]) {
+fn main(ops: [SignedDivOp; 15]) {
+    unsafe_div(ops);
+    test_div(ops);
+}
+
+unconstrained fn unsafe_div(ops: [SignedDivOp; 15]) {
+    test_div(ops);
+}
+
+fn test_div(ops: [SignedDivOp; 15]) {
```

Comment:
We automatically compile any ACIR programs in to brillig so we don't need to do this.

```suggestion
fn main(ops: [SignedDivOp; 15]) {
```